AUTHORS

```diff
@@ -0,0 +1,27 @@
+# Directly affiliated with DEDIS
+Bryan Ford <bryan.ford@epfl.ch>
+Kokoris Kogias Eleftherios <eleftherios.kokoriskogias@epfl.ch>
+Ismail Khoffi <ismail.khoffi@epfl.ch>
+Linus Gasser <linus.gasser@epfl.ch>
+Nicolas Gailly <nicolas.gailly@epfl.ch>
+Nikitin Kirill <kirill.nikitin@epfl.ch>
+Philipp Jovanovic <philipp.Jovanovic@epfl.ch>
+
```

Comment:
Nitpick: Shouldn't those names be ordered alphabetically either by <first name> <last name> <email> or <last name> <first name> <email>? Currently it's not consistent.